CHANGELOG.md

```diff
@@ -3354,6 +3376,11 @@ This is a full list of proofs that have changed form to use irrelevant instance
   ⊆-mergeʳ : ∀ xs ys → ys ⊆ merge _≤?_ xs ys
   ```
 
+* Added new proof to `Induction.WellFounded`
+  ```agda
+  Acc-resp-flip-≈ : _<_ Respectsʳ (flip _≈_) → (Acc _<_) Respects _≈_
```

Comment:
I took the liberty of changing the original qualifier in the name from `weak` to `flip`: hopefully more memorable!